pipelines/release/tarball.groovy

```diff
@@ -598,22 +598,32 @@ def void writeScript(Map p) {
  * Push {@code ./distrib} dir to an s3 bucket under the "path" formed by
  * joining the {@code parts} parameters.
  */
-def void s3PushDocker(String ... parts) {
+def void s3PushConda(String ... parts) {
   def objectPrefix = "stack/" + util.joinPath(parts)
   def cwd = pwd()
+  def buildDir = "${cwd}/build"
 
   def env = [
     "EUPS_PKGROOT=${cwd}/distrib",
     "EUPS_S3_OBJECT_PREFIX=${objectPrefix}",
     "HOME=${cwd}/home",
+    "BUILDDIR=${buildDir}"
   ]
 
   withEnv(env) {
     withEupsBucketEnv {
       timeout(time: 10, unit: 'MINUTES') {
-        docker.image(util.defaultAwscliImage()).inside {
+        //docker.image(util.defaultAwscliImage()).inside {
           // alpine does not include bash by default
-          util.posixSh(s3PushCmd())
+        util.posixSh("""
+        source "${BUILDDIR}/conda/miniconda3-py38_4.9.2/etc/profile.d/conda.sh"
```

Comment:
The `miniconda3-py38_4.9.2` part of this path might change in the future.  I think there should be a way to avoid hard-coding this, but we can create a ticket to fix that later.